setup.py

```diff
@@ -22,6 +18,16 @@
 extra_objects = mecab_config[3].split()
 extra_link_args = mecab_config[4].split()
 
+
+class build_ext(_build_ext):
+    def run(self):
+        if self.editable_mode and sys.platform == "win32":
+            fugashi_dir = pathlib.Path(__file__).parent / "fugashi"
+            for i in dll_files:
+                self.copy_file(i, fugashi_dir)
```

Comment:
The following modification will ensure that the dll is always installed:
```suggestion
-        if self.editable_mode and sys.platform == "win32":
-            fugashi_dir = pathlib.Path(__file__).parent / "fugashi"
+        if sys.platform == "win32":
+            if  self.editable_mode:
+                fugashi_dir = pathlib.Path(__file__).parent / "fugashi"
+            else:
+                fugashi_dir = pathlib.Path(self.build_lib) / "fugashi"
```
But if the dll is included in the wheel then delvewheel doesn't seem to handle it.
If the wheel contains the dll then delvewheel may seem unnecessary but it is necessary to avoid [DLL hell](https://en.wikipedia.org/wiki/DLL_hell).